app/helpers/fhir_helper.rb

```diff
@@ -241,6 +241,27 @@ def close_contact_from_fhir(related_person)
     }
   end
 
+  def history_as_fhir(history)
+    FHIR::Provenance.new(
+      meta: FHIR::Meta.new(lastUpdated: history.updated_at.strftime('%FT%T%:z')),
+      id: history.id,
+      patient_id: history.patient_id,
+      occurrenceDateTime: history.created_at,
+      recorded: history.updated_at,
+      reason: [
+        {
+          text: history.comment
+        }
+      ],
+      agent:[
+        {
+          who: FHIR::Reference.new(reference: "User/#{history.created_by}"),
+          onBehalfOf: FHIR::Reference.new(reference: "Patient/#{history.patient_id}")
```

Comment:
I think that makes perfect sense, though I would suggest possibly keeping onBehalfOf as well even if it is redundant, so that anyone writing scripts relying on our api could collect both the patient and the user references in a single item.

Reply:
Sounds good to me!